metricbeat/helper/sql/sql.go

```diff
@@ -167,24 +170,35 @@ func ReplaceUnderscores(ms mapstr.M) mapstr.M {
 }
 
 func getValue(pval *interface{}) interface{} {
-	switch v := (*pval).(type) {
-	case nil, bool:
-		return v
+	if pval == nil {
+		return nil
+	}
+
+	v := *pval
+
+	switch val := v.(type) {
+	case nil, bool, int64, uint64, float32, float64, string, []interface{}:
+		return val
+	case int:
+		return int64(val)
+	case uint:
+		return uint64(val)
+	case uint32:
+		return uint64(val)
```

Comment:
Did this as uint64 supported in numeric. See: https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html